packages/web/src/components/list/MultiDataList.js

```diff
@@ -424,6 +425,8 @@ class MultiDataList extends Component {
 
 		const listItems = this.listItems;
 
+		const isAllChecked = selectAllLabel ? !!this.state.currentValue[selectAllLabel] : null;
+
```

Comment:
Code like that is always confusing for the reader, why you're assigning `null` to a boolean variable.